src/object/srv_ec.c

```diff
@@ -160,6 +164,13 @@ obj_ec_rw_req_split(daos_unit_oid_t oid, struct obj_iod_array *iod_array,
 	tgt_oiod = obj_ec_tgt_oiod_get(tgt_oiods, count, leader);
 	D_ASSERT(tgt_oiod != NULL && tgt_oiod->oto_tgt_idx == leader);
 
+	if (tgt_oiod->oto_oiods->oiod_siods->siod_nr == 0) {
+		D_ERROR("lxz local io bypass\n");
+		req->osr_local_io_bypass = true;
+		//*split_req = req;
+		//goto out;
```

Comment:
(style)  Block comments use * on subsequent lines
(style)  do not use C99 // comments